### PR TITLE
ETC 7700 - 9500, Crystal Mines $NODATA fix

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -7728,18 +7728,18 @@ ETC_20150317_007728	Pick the herb asked by Cleric Master
...
 ETC_20150317_007736	{memo X}Sanctum of Secrecy - Archer type_conditions
 ETC_20150317_007737	{memo X}Sanctum of Secrecy - Cleric type_conditions
 ETC_20150317_007738	Purpose of the secret box
-ETC_20150317_007739	Passed Magician Owyn quiz
-ETC_20150317_007740	Answer the quiz of Dazzlementer Cordelier 
-ETC_20150317_007741	Answer the quiz of Transmuter Furryode
-ETC_20150317_007742	Collect a sheet from a wizard's book inside the laboratory on Mage's Tower 3rd Floor.
-ETC_20150317_007743	For fail and success
+ETC_20150317_007739	Passed Magician Owyn's quiz
+ETC_20150317_007740	Answer Cordelier's quiz
```
Comment (ttgmichael) [inline ETC.tsv:7739]: No Dazzlementer?

Comment (vtange) [inline ETC.tsv:7739]: In real Mage Tower she has no title. No other magician in there has "Dazzlementer" stuff.

She does have a "Bedazzlement" quest though

File: ETC.tsv [hunk 2]
```diff
@@ -8057,12 +8057,12 @@ ETC_20150317_008057	CAST Animation Speed
 ETC_20150317_008058	Attack Animation Speed
 ETC_20150317_008059	Channel Update Term
 ETC_20150317_008060	Monster's Chase Update Time
-ETC_20150317_008061	Maximum time of boss monster's skill chase. Target is removed when time is overdue.
+ETC_20150317_008061	Maximum time of boss monster's skill chase. Target is cancelled when overtime.
```
Comment (Sunaries) [inline ETC.tsv]: canceled (US English)

File: ETC.tsv [hunk 3]
```diff
@@ -8114,9 +8114,9 @@ ETC_20150317_008114	Kepa AI
 ETC_20150317_008115	Kepa AI2
 ETC_20150317_008116	Strike
 ETC_20150317_008117	Chapparition's Shadow
-ETC_20150317_008118	Vubbe Fighter is very exhausted.
-ETC_20150317_008119	Destroy the summoning post that Specter Monarch is casting
-ETC_20150317_008120	If it is not destroyed, the Monarch's lighting will stay continuous.
+ETC_20150317_008118	The Vubbe Fighter is very exhausted.
+ETC_20150317_008119	Destroy the summons the Specter Monarch summoned
```
Comment (Sunaries) [inline ETC.tsv]: The Spector Monach does summon this one summoning post. 'summons' implies multiple.

File: ETC.tsv [hunk 4]
```diff
@@ -8114,9 +8114,9 @@ ETC_20150317_008114	Kepa AI
...
+ETC_20150317_008120	If not, the Monarch's lightning will never end.
```
Comment (Sunaries) [inline ETC.tsv]: If not, the lightning will never end.

or

If not, the lightning from the Specter Monach will never end.

File: ETC.tsv (hunk 3 above)
Comment (stvkno) [inline ETC.tsv]: summons/summoning post sounds weird. even though the korean is summoned minion whatever it is is stationary and i think it should be totem.

Destroy the Specter Monarch's summoned totem.

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**